### ISSUE TITLE
No album cover art shows on local install

### ISSUE THREAD

**Issue by justalewis** [dr4g0nnn/VagrantGazelle#6]
Hi,

I installed VagrantGazelle for testing the WhatCD Android app. On my localhost:8080 install I don't get any image art in the "Cover" area of the torrent details page (see pic). I've tried both a URL as well as a local file location c:\ . . . "  

***UPDATE: I just tested personal profile avatars and they don't work either. Is there an image handler thing I should change? Sorry, I'm a noob! 

Any ideas of how to make this work? Thnx! 

justin
![gest](https://cloud.githubusercontent.com/assets/24281011/22348906/61d87332-e3db-11e6-9507-0f1ee7bc3e66.png)

**Comment by dr4g0nnn** [comment]
Sorry – I haven't used this in almost 4 years now, so it's a little hard to say.

In the first instance, I'd suggest taking a look at the `classes/config.template` in Gazelle and comparing it to the `config.php` here for anything that looks related, the one here will be woefully out of date.

**Comment by justalewis** [comment]
No worries. I didn't see any marked difference between the two (except, of course, the non template file was populated with details from the install I have here). 

Should I ask in the regular Gazelle repo (https://github.com/WhatCD/Gazelle/issues)? Sorry to be bothersome and thanks for your help.

**Comment by dr4g0nnn** [comment]
> No worries. I didn't see any marked difference between the two (except, of course, the non template file was populated with details from the install I have here).

Ah, that's a shame, I was hoping it would be something obvious.

> Should I ask in the regular Gazelle repo (https://github.com/WhatCD/Gazelle/issues)?

It's probably a better place for it, yes. If it does turn out to be a defect with VagrantGazelle, then let me know!

> Sorry to be bothersome and thanks for your help.

No trouble, sorry to not be of any help.

**Comment by justalewis** [comment]
Hi dr4g0nnn, 

After messing around a bit, I discovered that the problem is that all image handling requests end up rendering with "http://localhost/image.php?c=1&i=" as part of the URL address. 

This is directly followed by the actual URL "https%3A%2F%2Fs8.postimg.org%2Foac2yuplh%2Fglp.jpg". 

So, the image location is being defined by Gazelle as "http://localhost/image.php?c=1&i=https%3A%2F%2Fs8.postimg.org%2Foac2yuplh%2Fglp.jpg". 

Do you know if there's a place change an image setting to remove the "localhost/image.php?c=1&i=" portion of image handling? Thnx a ton -- I feel like this is an easy question to answer but I'm sadly underprepared to do so. :(

**Comment by dr4g0nnn** [comment]
> So, the image location is being defined by Gazelle as "http://localhost/image.php?c=1&i=https%3A%2F%2Fs8.postimg.org%2Foac2yuplh%2Fglp.jpg".
> 
> Do you know if there's a place change an image setting to remove the "localhost/image.php?c=1&i=" portion of image handling? Thnx a ton -- I feel like this is an easy question to answer but I'm sadly underprepared to do so. :(

It looks like the relevant code is here: [classes/imagetools.class.php:171](https://github.com/WhatCD/Gazelle/blob/master/classes/imagetools.class.php#L171)

The real question is why the image proxying isn't working for you. Looking at [image.php](https://github.com/WhatCD/Gazelle/blob/master/image.php), a likely candidate is that you don't have the `gd` extension for PHP installed, in which case you should be able to fix it by doing:

```
sudo apt-get install php5-gd
sudo /etc/init.d/apache2 restart
```

**Comment by justalewis** [comment]
Thnx for the message back, dr4g0nnn. I really appreciate it. 

I've installed php5-gd and restarted apache. 

The problem is definitely different now. Instead of a broken picture, I can now see only text but no image: https://s7.postimg.org/de1rw1h8r/better_image.png 

I'm going to have a look at the imagetools.class.php file #171 to see what's going on there. I noticed that there is some information about the added " image.php?c=1&i=" at the end of that section. 

Time to read some more. Thnx again for the help and any other information you may have is welcome!

**Comment by justalewis** [comment]
Thnx for the information regarding missing dependencies. I did manage to get this working (and the solution is simpler than it should have been -- again, I'm a noob here). 

I installed php5-gd from bash. Next, I entered "config.php" located in VagrantGazelle-->src-->classes. I changed the NONSSL-SITEURL & SSL-SITEURL to include the access port (changed from localhost to localhost:8080).

Saved everything and then rebooted vagrant gazelle ('vagrant up from command line). That did it. 

Thanks for all the help here - I really appreciate it!
